packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_webview_controller.dart

```diff
@@ -286,6 +286,36 @@ abstract class PlatformWebViewController extends PlatformInterface {
       'setOnConsoleMessage is not implemented on the current platform',
     );
   }
+
+  /// Sets a callback that notifies the host application that the web page
+  /// wants to display a JavaScript alert() dialog.
+  Future<void> setOnJavaScriptAlertDialog(
+      Future<void> Function(String message)
```

Comment:
It looks like [WebChromeClient.onJsAlert](https://developer.android.com/reference/android/webkit/WebChromeClient#onJsAlert(android.webkit.WebView,%20java.lang.String,%20java.lang.String,%20android.webkit.JsResult)) and [WKUIDelegate.runJavaScriptAlertPanelWithMessage](https://developer.apple.com/documentation/webkit/wkuidelegate/1537406-webview?language=objc) also provide the URL for these callbacks. We should change `String message` to a class that contains a `message` field where we can add additional information at a later point. You could create a class named something like `JavaScriptAlertDialogRequest`. See [JavaScriptConsoleMessage](https://github.com/flutter/packages/blob/main/packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/javascript_console_message.dart) as an example.

This goes for the methods below as well. (e.g. `JavaScriptConfirmDialogRequest` and `JavaScriptTextInputDialogRequest`).

Reply:
I fixed it. Can you check it again?